gaia-icons.css

```diff
@@ -11,9 +11,9 @@
 
 
 [data-icon]:before,
+[data-icon-end]:after,
```

Comment:
Is it also possible to add similar style for inaccessible icon as with:
[data-icon]:not([data-l10n-id]):not([aria-hidden="true"]):before
and its relevant message:
[data-icon]:not([data-l10n-id]):not([aria-hidden="true"]):after